src/utils/requestNavigationHandlers.ts

```diff
@@ -158,8 +157,7 @@ export const handleMobileWalletProtocolRequest = async ({
       }
 
       if (action.method === 'wallet_switchEthereumChain') {
-        const chainId = BigNumber.from(action.params.chainId).toNumber();
-        const isSupportedChain = Object.values(ChainId).includes(chainId);
+        const isSupportedChain = SUPPORTED_MAINNET_CHAIN_IDS.includes(BigNumber.from(action.params.chainId).toNumber());
```

Comment:
@walmat same here if you could take a look - now it's going from all chains to non-testnet chains

Reply:
Yeah again only going forward users won't be able to switch chains. Shouldn't have any negative impact.